### ISSUE TITLE
How to rotate an image?

### ISSUE THREAD

**Issue by arthur-xie** [hybridgroup/gocv#46]
When try the [example](https://courses.learnopencv.com/courses/227056/lectures/3575746) code in go(cpp -> golang) to rotate an image.
```cpp
M = cv::getRotationMatrix2D(center, rotationAngle, scale);
cout << M << endl;
// Rotate the source and store in result
cv::warpAffine(source, result, M, Size(source.cols, source.rows));
```
I cannot find the counter-part in gocv.

So, how to rotate an image in golang without ```cv::getRotationMatrix2D``` or ```cv::warpAffine```? Please help me, thank you.

**Comment by deadprogram** [comment]
Hello @arthur-xie you might want to look at the `dev` branch of GoCV. We will be including both of those functions in the next release:

https://github.com/hybridgroup/gocv/blob/dev/imgproc.go#L724-L765

Hope that helps.

**Comment by deadprogram** [comment]
This code was released as part of v0.8.0 so now closing. Please reopen if anything else is needed. Thank you.
